packages/sheets/src/commands/commands/__tests__/set-style.command.spec.ts

```diff
@@ -755,4 +760,53 @@ describe("Test commands used for updating cells' styles", () => {
             });
         });
     });
+
+    describe('set style with specific range', () => {
+        it('should use the correct unitId and subUnitId when range is provided', async () => {
+            const workbook = get(IUniverInstanceService).getCurrentUnitForType<Workbook>(UniverInstanceType.UNIVER_SHEET)!;
+            if (!workbook) throw new Error('Workbook not found');
```

Comment:
It is not necessary to do this kind of error handling in unit tests. If `workbook` is a false value, this test would fail anyway.